kinto_fxa/authentication.py

```diff
@@ -60,7 +63,13 @@ def unauthenticated_userid(self, request):
             return None
         if authmeth.lower() != 'bearer':
             return None
-        return self._verify_token(token, request)
+
+        user_id, client_name = self._verify_token(token, request)
+
```

Comment:
`Don't add suffix if authentication failed, or no specific client name is configured`

Reply:
Is it a comment you want to add?

Reply:
Yes for the `return user_id` below